src/test/java/au/com/codeka/carrot/CarrotEngineTest.java

```diff
@@ -49,21 +49,28 @@ public void testEchoTag() {
   @Test
   public void testAutoEscape() {
     CarrotEngine engine = createEngine();
-    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", Maps.newHashMap()))
+    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", new HashMap<String, Object>()))
         .isEqualTo("Some &lt;b&gt;HTML&lt;/b&gt; here");
 
     engine.getConfig().setAutoEscape(false);
-    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", Maps.newHashMap()))
+    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", new HashMap<String, Object>()))
         .isEqualTo("Some <b>HTML</b> here");
 
     engine.getConfig().setAutoEscape(true);
-    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", Maps.newHashMap()))
+    assertThat(render(engine, "{{ \"Some <b>HTML</b> here\" }}", new HashMap<String, Object>()))
         .isEqualTo("Some &lt;b&gt;HTML&lt;/b&gt; here");
   }
 
   private CarrotEngine createEngine() {
     CarrotEngine engine = new CarrotEngine();
-    engine.getConfig().setLogger((level, msg) -> System.err.println(msg));
+    engine.getConfig().setLogger(new Configuration.Logger()
+    {
```

Comment:
I normally prefer the K&R/Java style of brackets on the line above, but I'll not worry about it this time. Just something for next time :)